src/main/java/com/conveyal/r5/profile/Propagater.java

```diff
@@ -56,8 +60,12 @@ public Propagater (int[][] travelTimesToStopsEachIteration, int[] nonTransferTra
         // iteration to the next, but that requires working on clock times, so we'd have to pass more information down
         // into this function; I'm not sure it's worth it.
         int[] result = new int[travelTimesToStopsEachIteration.length];
+        int[] travelTimesToTargetsThisIteration = new int[nonTransferTravelTimesToTargets.length];
```

Comment:
This would be more readable if we pulled nIterations and nTargets out into named final int variables and used those to specify the array dimensions. Likewise below in the loop termination conditions.